src/commands/furry/e621.js

```diff
@@ -0,0 +1,28 @@
+const { Command, SwitchbladeEmbed, Constants } = require('../../')
+const booru = require('booru')
+
+module.exports = class E621 extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'e621'
+    this.aliases = ['yiff']
+  }
+
+  async run (message) {
+    message.channel.startTyping()
+    const embed = new SwitchbladeEmbed(message.author)
+    if (message.channel.nsfw) {
+      const images = await booru.search('e621.net', ['rating:e'], {limit: 1, random: true})
+        .then(booru.commonfy)
+      for (let image of images) {
+        embed.setImage(image.common.file_url)
+        embed.setDescription('Here\'s your yiff!')

```

Comment:
```js
const [ image ] = await booru.search('e621.net', ['rating:e'], {limit: 1, random: true}).then(booru.commonfy)
embed.setImage(image.common.file_url)
embed.setDescription('Here\'s your yiff!')```

Reply:
it doesn't works 
![](http://idonthave.life/i/5f3iP.png)

Reply:
Same for your english

Reply:
define that

Reply:
ignore, the embeds was glitched